third_party/cargo/remote/BUILD.indicatif-0.16.2.bazel

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
Are all of these going to cause problems if we enable license checks for *.bazel files? Should we add an exception for third_party/cargo in anticipation of that?

Reply:
Created #9361